js/payment-methods/monero.js

```diff
@@ -23,6 +27,7 @@ app.paymentMethods.monero = (function() {
 				'settings.public-address.invalid': 'Invalid public address',
 				'settings.view-private-key.label': 'View Private Key',
 				'settings.view-private-key.invalid': 'Invalid view private key',
+				'payment-request.public-address-required': 'Public address is required to generate a payment request',
```

Comment:
Why is this here, since you seem to be using it at line 84 using `i18n`?

Reply:
The language strings for payment methods are prefixed by the method's key/name/reference (ie. `monero` here). \